src/OpenTelemetry.Instrumentation.Wcf/README.md

```diff
@@ -224,6 +224,12 @@ on the service contracts you want to instrument:
     }
 ```
 
+### RecordException
```

Comment:
Seems to be missing a header/section. Here is a suggestion, adapted from SqlClient README:

## Advanced configuration

This instrumentation can be configured to change the default behavior by using
`WcfInstrumentationOptions`.